docs/Toy/toy/frontend/parser.py

```diff
@@ -23,6 +23,12 @@
 )
 
 
+def loc(token: ToyToken):
+    file = token.span.input.name
+    line, col = token.span.get_line_col()
+    return Location(file, line, col + 1)
```

Comment:
I'm not sure why there's a disagreement between toy locations and our span locations...

Reply:
I have a feeling that it's our span locations that are wrong, and that columns universally start at 1.